### DISCUSSION TITLE
ipv6 socket/connect support

### DISCUSSION THREAD

**Discussion by d1115choi** [lldpd/lldpd#603] [Q&A]
Hello,

I am wondering if there is any lldp branch to support ipv6. The error message is not shown while running lldpd in ipv4 configured system. But the when exact same system is configured only in ipv6, I see the message below, which I think causes not to communicate between lldpcli and lldpd. Is there any solution or how do I resolve it?
Thank you in advance for the help.

 [WARN/control] unable to connect to socket /tmp/var/lldpd.socket

**Comment by vincentbernat** [comment]
This is a Unix socket and it does not require IPv4. The error message should also display the system error returned.

**Comment by d1115choi** [comment]
Thank you Vincentbernat for your reply. What you said is that it is nothing to do with ipv4 or ipv6 as it is Unix socket?
Let me dig the issue and update you when I get something new.
